amplify/backend/api/themeetinghouse/schema.graphql

```diff
@@ -11,6 +11,7 @@ type Query {
   getFBEvents(pageId:String):FBData
   getTakenoteNotes(sermonId:String,noteType:String):TakeNoteNotes
   fuzzySearchVideos(filter: String, sort: fuzzySearchableVideoSortInput, limit: Int, nextToken: String):fuzzySearchableVideoConnection
+  fuzzySearchVideosByType(videoTypes: String, filter: String, limit: Int, nextToken: String):fuzzySearchVideosByTypeConnection
```

Comment:
Adding this to the GraphQL requires 
- changes to amplify/backend/api/themeetinghouse/stacks/CustomResources.json
- associated req.vtl and res.vtl files in amplify/backend/api/themeetinghouse/resolvers 

We may also need to do something to the ElasticSearch instance...but likely it has all the data you need, just need to update the query in the req.vtl file.

Once you get all that done I will push it to the beta site...assuming that works - I'll push it to production as it will have no breaking changes.

If we get into breaking changes we can see about getting you access to push to beta...